code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -236,16 +236,58 @@
 	list_reagents = list("wine" = 100)
 
 /obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe
-	name = "T&T Extra-Strong Absinthe"
-	desc = "An strong alcoholic drink brewed and distributed by Terb & Tonte Inc."
+	name = "Extra-Strong Absinthe"
+	desc = "An strong alcoholic drink brewed and distributed by FULL_COMPANY Inc."
 	icon_state = "absinthebottle"
 	list_reagents = list("absinthe" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe/New()
+	..()
+	redact()
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe/proc/redact()
+	// There was a large fight in the coderbus about a player reference
+	// in absinthe. Ergo, this is why the name generation is now so
+	// complicated. Judge us kindly.
+	var/shortname = pickweight(
+		list("T&T" = 1, "A&A" = 1, "Generic" = 1))
+	var/fullname
+	switch(shortname)
+		if("T&T")
+			fullname = "Teal and Tealer"
+		if("A&A")
+			fullname = "Ash and Asher"
+		if("Generic")
+			fullname = "Nanotrasen Cheap Imitations"
+	var/removals = list("\[REDACTED\]", "\[EXPLETIVE DELETED\]",
+		"\[EXPUNGED\]", "\[INFORMATION ABOVE YOUR SECURITY CLEARANCE\]",
+		"\[MOVE ALONG CITIZEN\]", "\[NOTHING TO SEE HERE\]")
+	var/chance = 50
```

Comment:
It doesn't really matter but why make this a var? It's only used twice and isn't changed by anything.